src/cata_tiles.cpp

```diff
@@ -518,6 +519,28 @@ void cata_tiles::load_tilejson_from_file(const std::string &tileset_dir, std::if
         load_tilejson_from_file(config, 0, newsize);
         offset = newsize;
     }
+
+    // allows a tileset to override the order of mutation images being applied to a character
+    if( config.has_array( "overlay_ordering" ) ) {
+        tileset_mutation_overlay_ordering.clear();
```

Comment:
This should probably outside of the branch. It's currently not reset when loading a tileset that does not have that JSON member and therefor remains in the state of *previous* tilset, not the default.